clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
+    "description": "Typically used to communicate between Java/C/C++/Python/etc. and COBOL, with COMP-5, data, the size of the variable is defined by the field's PIC size. When numeric data is moved into a COMP-5 item, truncation occurs at the binary field size rather than at the COBOL picture clause declaration. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMPUTE": {
+    "prefix": "COMPUTE",
+    "body": [
+      "COMPUTE ${1:NUM-VAR} OF ${2:DFHCOMMAREA} = ${3:WS-NUMVAR1} ${4|+,-,*,/,**|} ${5:WS-NUMVAR2}",
+      "\tON SIZE ERROR GO TO ${6:TRAP-OVERFLOW}",
+      "\tNOT ON SIZE ERROR GO TO ${7:PROGRAM-FINAL-ACTION}",
+      "END-COMPUTE",
+      "$0"
+    ],
+    "description": "COMPUTE provides an alebraic-style keyword for the speficiation of complex arithemetic expressions: + (add), - (subtract), * (multiply), / (divide), ** (power of) and any intrinsic functions that return numeric results. ON SIZE ERROR allows you to trap situcations such as divide by zero and result overflow. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=statements-compute-statement"
+  },
   "COMPUTE EQUAL": {
     "prefix": "COMPUTE EQUAL",
     "body": ["COMPUTE ${1:id} EQUAL ${2:arithExp}"],
     "description": "COMPUTE id EQUAL arithexp"
   },
+  "CONFIGURATION": {
+    "prefix": "CONFIGURATION",
+    "body": ["ENVIRONMENT DIVISION.", "CONFIGURATION SECTION."],
+    "description": "The CONFIGURATION SECTION describes the z/OS environment on which the program is compiled and executed. It is an optional section for programs, functions, prototypes, and classes. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=division-configuration-section"
+  },
+  "CONTAINS": {
+    "prefix": "CONTAINS",
+    "body": [
+      "FD ${2:inputFile}",
+      "\tRECORD CONTAINS ${1:numLiteral} CHARACTERS",
+      "\tBLOCK CONTAINS 0 RECORDS",
+      "\tDATA RECORD IS ${3:inputFileRec}."
+    ],
+    "description": "A fixed-block file. Note that BLOCK CONTAINS 0 leaves the blocking-optimization up to z/OS - and is the preferred way to specify most QSAM file FDs. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=entries-block-contains-clause"
+  },
   "CONTENT": {
     "prefix": "CONTENT",
     "body": ["CONTENT ${1:id}"],
     "description": "CONTENT id"
   },
+  "CONTENT-OF": {
+    "prefix": "CONTENT-OF",
+    "body": [
+      "COMPUTE ${1:resultNum} = CALC2(CONTENT-OF(${2:arg-1}),${3:arg-2},${4:arg-3})",
+      "END-COMPUTE"
+    ],
+    "description": "The CONTENT-OF intrinsic function produces a temporary copy of the input argument. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=functions-content"
+  },
+  "CONTROL": {
+    "prefix": "CONTROL",
+    "body": ["*CONTROL ${1|MAP,NOMAP|)"],
+    "description": "*CONTROL is a Compiler option - that allows the user to limit (or not limit) the MAP output of a program compile.  https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf "
```

Comment:
"*CONTROL is a compiler option which allows the user to limit (or not limit) the MAP output of a program compile.  https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf "